metrics/config/metrics.go

```diff
@@ -1,10 +1,12 @@
 package config
 
 import (
+	"log/slog"
```

Comment:
Please continue to use glog for this PR. We'll discuss changing logging libraries in a separate thread.

Reply:
ok - sounds good